src/test/java/org/dependencytrack/resources/v1/ConfigPropertyResourceTest.java

```diff
@@ -383,4 +384,17 @@ public void updateConfigPropertyBomValidationTagsInclusiveTest() {
         assertThat(getPlainTextBody(response)).isEqualTo("Value must be a valid JSON array of strings");
     }
 
+    @Test
+    public void getPublicForbiddenTest() {
```

Comment:
A test for the success-case would be good. You could even loop over all values of `ConfigPropertyConstants` to verify the desired behavior for all of them.